samples/nrf5340/netboot/README.rst

```diff
@@ -1,88 +1,95 @@
+* If you don't need the capability to update the sample, enable the :option:`CONFIG_SECURE_BOOT` option in the application that runs on the network core.
+* If you need the capability to update the sample, enable the :option:`CONFIG_BOOTLOADER_MCUBOOT` option in the application that runs on the application core to add MCUboot to the build.
+  When doing so, the build system includes the sample in the build by enabling automatically the :option:`CONFIG_SECURE_BOOT` option for the application that runs on the network core.
+  In addition, you must also enable the :ref:`subsys_pcd` library for MCUboot by setting the :option:`CONFIG_PCD` option when building its image.
+  See :ref:`ug_multi_image` for more information about setting configurations to a specific image.
 
-A new set of firmware update files is generated by the build system when this image is included in a multi-image build.
-These files are identical to those described in :ref:`mcuboot:mcuboot_ncs`, except that they contain the network core application firmware and are prefixed with ``net_core_``.
+  .. note::
+     In this setup, MCUboot verifies any new image received through a device firmware update (DFU) transport layer, like a serial or a Bluetooth LE connection, for both the application and the network core.
+     This includes the network core bootloader image.
+     See :ref:`subsys_pcd` for more details.
 
-Testing
-=======
+When you include this sample in a multi-image build, the build system generates a new set of firmware update files.
+These files match the ones described in :ref:`mcuboot:mcuboot_ncs`, except that they contain the network core application firmware and are prefixed with ``net_core_``.
 
-To test the network core bootloader sample run the following commands:
+This sample can be found under :file:`samples/nrf5340/netboot/` in the |NCS| folder structure.
 
-#. |connect_terminal|
+See :ref:`configure_application` for information on how to enable the required configuration options.
+Then follow the instructions in :ref:`ug_nrf5340_building` to build and program the images for the network and application core.
 
-   Note that on the nRF5340 DK has multiple UART instances, so the correct port must be identified.
+.. note::
+   To try out the network core bootloader sample, you can use the :ref:`peripheral_uart` sample as the basis for the multi-image build.
+   This sample automatically includes the network core sample :ref:`bluetooth-hci-rpmsg-sample` when built for the nRF5340 DK.
+   You can then apply the options mentioned above to include the network core sample with or without MCUboot.
 
-#. ``west build -b nrf5340dk_nrf5340_cpuapp -d build_netboot samples/bluetooth/peripheral_uart -f -- -DCONFIG_BOOTLOADER_MCUBOOT=y -Dmcuboot_CONFIG_PCD=y``
+Testing
```

Comment:
We should have the build instructions in place both for command line and SES

Reply:
@hakonfam the whole scope of the JIRA that made this PR necessary was to actually make the sample docs mention the official way we build samples in NCS, through SES. The instructions are linking to the pages that address this. In a previous version of this doc I realized it would be just a copypaste of these, so to remove any duplication I linked them there like it is done in most of the other NCS samples.